README.md

```diff
@@ -1 +1,9 @@
+- Project-1 of (FreeCodeCamp) FCC of building cat photo app.
 
+- Project-2 of FCC (FreeCodeCamp) for learning basic CSS by building a Cafe Menu! It's a responsive web-design of FCC (FreeCodeCamp).
+
+- Project-3 of FCC (FreeCodeCamp) of Basic Concept Of (CSS Color Markers) From FCC (FreeCampCode) such as : RGB , HEX , HSL . From this I learnt great concepts about color markers and how to add a perfect gradient color and many others.
+
+- Project-4 of FCC (FreeCodeCamp) of making a basic registration form. Very easy and great form easy to learn!
+
+- Project-5 of FreeCodeCamp for Rothko Painting. I learn html and css which is very helpful for me.
```

Comment:
Please add more details about project 4 and 5. E.g I developed _this and this_ using _this and that_

Reply:
Purpose of this README is to explain WHAT we built, HOW we built and WHAT we learnt.